sdk/python/feast/telemetry.py

```diff
@@ -45,35 +47,38 @@ def check_env_and_configure(self):
             self._telemetry_enabled = telemetry_enabled
 
             if self._telemetry_enabled:
-                feast_home_dir = join(expanduser("~"), ".feast")
-                Path(feast_home_dir).mkdir(exist_ok=True)
-                telemetry_filepath = join(feast_home_dir, "telemetry")
-
-                self._is_test = os.getenv("FEAST_IS_TELEMETRY_TEST", "False") == "True"
-                self._telemetry_counter = {"get_online_features": 0}
-
-                if os.path.exists(telemetry_filepath):
-                    with open(telemetry_filepath, "r") as f:
-                        self._telemetry_id = f.read()
-                else:
-                    self._telemetry_id = str(uuid.uuid4())
+                try:
+                    feast_home_dir = join(expanduser("~"), ".feast")
+                    Path(feast_home_dir).mkdir(exist_ok=True)
+                    telemetry_filepath = join(feast_home_dir, "telemetry")
 
-                    with open(telemetry_filepath, "w") as f:
-                        f.write(self._telemetry_id)
-                    print(
-                        "Feast is an open source project that collects anonymized error reporting and usage statistics. To opt out or learn"
-                        " more see https://docs.feast.dev/reference/telemetry"
+                    self._is_test = (
+                        os.getenv("FEAST_IS_TELEMETRY_TEST", "False") == "True"
                     )
+                    self._telemetry_counter = {"get_online_features": 0}
+
+                    if os.path.exists(telemetry_filepath):
+                        with open(telemetry_filepath, "r") as f:
+                            self._telemetry_id = f.read()
+                    else:
+                        self._telemetry_id = str(uuid.uuid4())
+
+                        with open(telemetry_filepath, "w") as f:
+                            f.write(self._telemetry_id)
+                        print(
+                            "Feast is an open source project that collects anonymized error reporting and usage statistics. To opt out or learn"
+                            " more see https://docs.feast.dev/reference/telemetry"
+                        )
+                except Exception as e:
+                    _logger.debug("Unable to configure telemetry", e)
```

Comment:
Is logging disabled by default? Ideally we would not print logs relating to Telemetry to users